ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -143,23 +145,23 @@ final class SandboxServer(
 
   private[this] val engine = {
     val engineConfig = {
-      val baseConfig =
-        if (config.seeding.isEmpty) {
-          if (config.devMode) {
+      val allowedLanguageVersions =
+        config.engineMode match {
+          case EngineMode.Stable if config.seeding.nonEmpty =>
+            LanguageVersion.StableVersions
+          case EngineMode.Stable =>
+            LanguageVersion.LegacyVersions
+          case EngineMode.EarlyAccess if config.seeding.nonEmpty =>
+            LanguageVersion.EarlyAccessVersions
+          case EngineMode.Dev if config.seeding.nonEmpty =>
```

Comment:
Is the "if" clause intended in this place

Reply:
Yes. This correspond to the case

Reply:
Yes. 
Since LF 1.dev is not designed to works with legacy Contract ID, I think it is safer to not start the sandbox at all in this case.
But if you prefer dropping the condition, it is fine with me.

Reply:
No that's fine. We have a catch-all clause further down. I just wanted to make sure it is intended.